### PR TITLE
AboutMe

### PR DESCRIPTION
**Pull request by srf6413**

### CODE REVIEW

**Comment by srf6413** [comment]
Pease disregard the first two pushes regarding StatsPageServlet

**Comment by ctrokid** [comment]
Thanks for opening this, Saulo!

Do you see that the "all checks have failed" warning is apparent? I think you should read through the `README.md` in the project, and hook up the checkstyle linter as your github pre-commit check (should be detailed in the readme - let me know if you need help).

After that, whenever you run `git commit -m "my message"`, the linter will check your style before allowing you to commit.

I will be happy to review this after the style issues are fixed. That way, I won't duplicate any comments that the linter will already warn you about.

**Comment by srf6413** [comment]
Ok thanks Collin! I didn’t know exactly what to do about those errors so thanks for clarifying!

**Comment by srf6413** [comment]
Should be good to go!

File: src/main/webapp/js/user-page-loader.js [hunk 1]
```diff
@@ -95,4 +114,5 @@ function buildUI() {
   setPageTitle();
   showMessageFormIfViewingSelf();
   fetchMessages();
+  fetchAboutMe();
 }
```
Comment (ctrokid) [inline src/main/webapp/js/user-page-loader.js:118]: Nitpicky: newline at the end of the file

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    String userEmail = userService.getCurrentUser().getEmail();
+
+    System.out.println("Saving about me for " + userEmail);
+    // TODO: save the data
+
+    response.sendRedirect("/user-page.html?user=" + userEmail);
+  }
+
+}
```
Comment (ctrokid) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: Same nitpicky comment 😆

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    UserService userService = UserServiceFactory.getUserService();
+    if (!userService.isUserLoggedIn()) {
+      response.sendRedirect("/index.html");
+      return;
+    }
+
+    String userEmail = userService.getCurrentUser().getEmail();
+
+    System.out.println("Saving about me for " + userEmail);
+    // TODO: save the data
```
Comment (ctrokid) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: When or who will do this?

My team at work has a TODO(bug/ID) format or TODO(username) in order to provide more context for the TODO.

**Review by ctrokid — COMMENTED** [review]
This looks good Saulo! A few small things and then I will approve.

Also, one thing, could you change the base branch from `master` to `develop` before merging?

**Comment by srf6413** [comment]
Ok I changed the base and it now says review required again

**Review by ctrokid — APPROVED** [review]
LGTM should be good to go now. I am approving.

**Pull request merged**